default/methods/source_control/file/file_browser.py

```diff
@@ -675,6 +723,7 @@ def file_view_core(
                 return False
             file_count += count
         else:
+            # TODO: I think we can remove this now?
```

Comment:
I believe this policy check can be removed now, but not sure if anywhere else in code needs it